index.html

```diff
@@ -53,7 +53,7 @@ <h1 class="generic_title">I’m Eugene Glad to see you!</h1>
       <section class="works-wrapper">
         <article class="work-card">
           <div class="work-card_picture">
-            <img src="./img/posts/tonic1.png" alt="Tonic" />
+            <img src="./img/posts/tonic1.png" alt="Tonic" alt="tonic1"/>
```

Comment:
- [ ] Great job deploying your portfolio on GitHub and including the link in your README.md file, but the images are not coming up, please see a screenshot below:

![WhatsApp Image 2022-12-16 at 10 45 19](https://user-images.githubusercontent.com/78151836/208072663-672151ec-e3b2-45c5-8a8b-b4909f620807.jpg)
consider using a relative instead of an absolute path, your image tag should look like `<img src="img/posts/tonic1.png" alt="Tonic" />` You can get more information on relative and absolute paths on this [link](https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/)